rdrf/rdrf/management/commands/migrate_gridfs_django.py

```diff
@@ -0,0 +1,183 @@
+from collections import namedtuple, OrderedDict
+from tempfile import TemporaryFile
+from shutil import copyfileobj
+import hashlib
+import logging
+import gridfs
+from django.core.management.base import BaseCommand
+from django.core.files import File
+from ...models import Registry, RegistryForm, Section, CommonDataElement, CDEFile
+from ...filestorage import FileStoreUtil
+from ...mongo_client import construct_mongo_client
+from ...utils import mongo_db_name
+
+logger = logging.getLogger("registry_log")
+
+
+class Command(BaseCommand):
+    help = "Transfer files across from GridFS into Django storage"
+
+    def add_arguments(self, parser):
+        parser.add_argument("--dry-run", action="store_true",
+                            help="Don't do anything, just log.")
+        parser.add_argument("registry", nargs="*")
+
+    def handle(self, *args, **options):
+        all_codes = Registry.objects.values_list("code", flat=True)
+        for registry_code in options["registry"]:
+            if registry_code not in all_codes:
+                logger.warning("Registry \"%s\" does not exist." % registry_code)
```

Comment:
'Registry "%s" does not exist.' % registry_code
reads better IMHO. PEP-8 also says so.